samples/get_batch_prediction_job_sample.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_get_batch_prediction_job_sample]
+from google.cloud import aiplatform
+from google.protobuf import json_format
+
+
+def get_batch_prediction_job_sample(project: str, batch_prediction_job_id: str):
+    client_options = dict(api_endpoint="us-central1-aiplatform.googleapis.com")
+    client = aiplatform.JobServiceClient(client_options=client_options)
+    location = "us-central1"
+    name = client.batch_prediction_job_path(
+        project=project, location=location, batch_prediction_job=batch_prediction_job_id
+    )
+    response = client.get_batch_prediction_job(name=name)
+    print("response")
+    print(" name:", response.name)
```

Comment:
Where possible, f-strings are preferred: 
```suggestion
    print(f" {response.name}:")
```